docs/interactions/structured-messages/apple-auth.md

```diff
@@ -30,18 +33,73 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 |-|-|-|
 | **`source_id`** | String | ID of the source. |
 | **`in_reply_to_id`** | String | ID of the message being replied to. |
-| **`body`** | String | The authenticate structured content body. |
+| **`body`** | String | Content of the authentication message that the customer will receive. Optional if you provided the "Default title" field with a value in your channel settings. |
 | **`structured_content`** | Object | Payload of the structured message. |
 | **Structured Content Settings** | | |
 | **`structured_content.type`** | String | Type of the structured message. Must be "authenticate". |
-| **`structured_content.response_body`** | String | **Optional**. Field to be the body of the user’s response. If absent, default is the structured message body. |
+| **`structured_content.response_body`** | String | **Optional**. Field to be the body of the user’s response. If absent, the default is the structured message body. |
```

Comment:
![Uploading Screen Shot 2022-12-21 at 17.22.28.png…]()

Reply:
looks like the image is broken

Reply:
<img width="1276" alt="Screen Shot 2022-12-21 at 17 22 28" src="https://user-images.githubusercontent.com/1233351/209345330-ebbf7644-ce2b-4b29-9f06-b605877c175e.png">